utils/helpers/coreHelper.ts

```diff
@@ -3,6 +3,7 @@ import * as setProtocolUtils from 'set-protocol-utils';
 import { Address } from 'set-protocol-utils';
 
 import {
+  AddressToAddressWhiteListContract,
```

Comment:
Since this isn't related to "Core", I would suggest putting this in a separate helper - like libraryHelper or utilsHelper

Reply:
👍